main_objs.py

```diff
@@ -637,7 +660,12 @@ def getNgrams(self, df=None, n=3, how='columnwise', other=None, held='Held',
                         mask = col.apply(lambda cell: all([excl not in cell for excl in exclude]))
                         col = col[mask]
                     cols.append(col)
-            return pd.concat(cols, axis=1)
+
+            # TODO in case piece has no harmony and cols stays empty
```

Comment:
Remove "TODO". Is this for a bug that came up in single-voice pieces? Awesome if so! Ideally this would have been in a separate commit so that it can have a commit string saying what it's fixing. It's fine as it is this time though.